util/rbac/rbac.go

```diff
@@ -97,6 +97,16 @@ func (e *Enforcer) EnforceErr(rvals ...interface{}) error {
 			for i, rval := range rvals[1:] {
 				rvalsStrs[i] = fmt.Sprintf("%s", rval)
 			}
+			switch s := rvals[0].(type) {
+			case jwt.Claims:
+				claims := s.(*jwt.StandardClaims)
+				if claims.Subject != "" {
+					rvalsStrs = append(rvalsStrs, fmt.Sprintf("sub: %s", claims.Subject))
+				}
+				if claims.IssuedAt != 0 {
+					rvalsStrs = append(rvalsStrs, fmt.Sprintf("iat: %d", claims.IssuedAt))
```

Comment:
Maybe we should print claims.IssuedAt as `time.Unix(t.IssuedAt, 0).Format(time.RFC3339)`

Reply:
Already addressed, thanks May!